### PR TITLE
SvelteKit release candidate migration: about page

### PR DESCRIPTION
**Pull request by EthanThatOneKid**
- Copied over the required Svelte for the about page
- Refactored repeated code
- Resolved TypeScript errors

### CODE REVIEW

File: src/lib/public/board/data/tiers.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[
```
Comment (k4rni) [inline src/lib/public/board/data/tiers.json:1]: Tiers need to be updated (src/lib/public/board/data/officers.json should reflect these tiers)

File: src/lib/public/board/utils.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import type { Officer, Term } from './types';
+import { TIERS, VISIBLE_TERMS } from './data';
+import { writable } from 'svelte/store';
+
+export const termIndex = writable<number>(0);
+
+export function getPositionByTermIndex(
+  officer: Officer,
+  termIndex: number
```
Comment (k4rni) [inline src/lib/public/board/utils.ts:9]: Rename this to "index" to avoid conflicts with termIndex variable name

File: tests/test.ts [hunk 3]
```diff
@@ -4,3 +4,8 @@ test('index page has expected h1', async ({ page }) => {
   await page.goto('/');
   expect(await page.textContent('h1')).toContain('We are the largest computer science community');
 });
+
```
Comment (k4rni) [inline tests/test.ts:7]: no way this is so cool

**Review by k4rni — CHANGES_REQUESTED** [review]
Great start:) Now fix it.

File: src/lib/public/board/utils.ts (hunk 2 above)
Comment (EthanThatOneKid) [reply src/lib/public/board/utils.ts:9]: This nit can be addressed in a future refactor.

File: src/lib/public/board/data/tiers.json (hunk 1 above)
Comment (EthanThatOneKid) [reply src/lib/public/board/data/tiers.json:1]: The tiers indeed must be updated. This can be done in a future PR.

File: tests/test.ts (hunk 3 above)
Comment (EthanThatOneKid) [reply tests/test.ts:7]: Playwright tests are great! Related: #557.

**Review by k4rni — APPROVED** [review]
This looks very sexy

**Comment by EthanThatOneKid** [comment]
Replaced by mega-PR #577.

**Pull request closed without merging**